internal/venice-common/src/main/java/com/linkedin/venice/controllerapi/ControllerRoute.java

```diff
@@ -134,6 +135,9 @@ public enum ControllerRoute {
   ROLLBACK_TO_BACKUP_VERSION(
       "/rollback_to_backup_version", HttpMethod.POST, Collections.singletonList(NAME), REGIONS_FILTER
   ),
+  AGGREGATED_HEALTH_STATUS(
+      "/aggregatedHealthStatus", HttpMethod.POST, Collections.emptyList(), AGGR_HEALTH_STATUS_PARAM
+  ),
```

Comment:
Can we also and one integration test that doesn't use `ControllerClient`, but rather some basic HTTP call?